themes/kpn.css

```diff
@@ -39,7 +39,7 @@ body {
   background-color: #FFFFFF; }
 
 .reveal {
-  font-family: "KPN Metric Light", "Lato", sans-serif;
+  font-family: "Segoe UI", "Lato", sans-serif;
```

Comment:
Note this is not the proper KPN font.

Reply:
But it *is* widely available to viewers.